src/wallet/feebumper.cpp

```diff
@@ -284,7 +290,7 @@ bool CFeeBumper::commit(CWallet *pWallet)
         // along with an exception. It would be good to return information about
         // wtxBumped to the caller even if marking the original transaction
         // replaced does not succeed for some reason.
-        vErrors.push_back("Error: Created new bumpfee transaction but could not mark the original transaction as replaced.");
+        vErrors.push_back("Error: Created new bumpfee transaction but could not mark the original transaction as replaced");
```

Comment:
BTW, is there a reason to have these `Error: ` prefixes?

Reply:
Good point. Seems to be used only in a few places. I am going to remove the `Error:` prefix in the 3 locations it's used.